### PR TITLE
WIP [Content] Adds content and structure for site.

### PR DESCRIPTION
**Pull request by MikeBallan**

### CODE REVIEW

**Comment by MikeBallan** [comment]
@catchamonkey Not sure why Trevor isnt happy with me :(

**Comment by MikeBallan** [comment]
But other than the erro that i think everything is in place other than some copy editing

**Comment by catchamonkey** [comment]
@MikeBallan it's failing as you have a test for the structure controller, but don't have a structure controller, so just delete the file

```
tests/unit/controllers/structure-test.js
```

Also, shouldn't the favicon file be in the public root, not in img?

File: app/routes/prototype-avatar.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+import ScrollToTop from '../mixins/scroll-to-top';
+
+export default Ember.Route.extend(ScrollToTop, {
+  setupController(controller) {
+    controller.set('headerTitle', 'Avatar');
+    controller.set('headerCopy', 'Lorem ipsum dolor sit amet, consectetur adipisicing elit.');
```
Comment (catchamonkey) [inline app/routes/prototype-avatar.js]: any lorem throughout needs updating

File: app/routes/prototype-qanda-data-tables.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+import ScrollToTop from '../mixins/scroll-to-top';
+
+export default Ember.Route.extend(ScrollToTop, {
+  setupController(controller) {
+    controller.set('headerTitle', 'Q&A Data Tables');
+    controller.set('headerCopy', 'A simple approche to styling a data table for the use of Q&As.');
```
Comment (catchamonkey) [inline app/routes/prototype-qanda-data-tables.js]: typo approach

File: app/routes/prototype-stepped-list.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+import ScrollToTop from '../mixins/scroll-to-top';
+
+export default Ember.Route.extend(ScrollToTop, {
+  setupController(controller) {
+    controller.set('headerTitle', 'Stepped list');
+    controller.set('headerCopy', 'Nice and simple ordered list stlyed to look like a forms progress. All you need to do is add the class .active to the relivant li and the rest is handled by the CSS.');
```
Comment (catchamonkey) [inline app/routes/prototype-stepped-list.js]: typo styled

Comment (catchamonkey) [inline app/routes/prototype-stepped-list.js]: type relevant

File: app/routes/prototypes.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+import ScrollToTop from '../mixins/scroll-to-top';
+
+export default Ember.Route.extend(ScrollToTop, {
+  setupController(controller) {
+    controller.set('headerTitle', 'Prototype');
+    controller.set('headerCopy', 'Here you can see some of the things weve been playing around with that might one day be a part of Amaziums core.');
```
Comment (catchamonkey) [inline app/routes/prototypes.js]: Amazium's core

**Pull request merged**